### ISSUE TITLE
Feature: Show menu path for commands you search for

### ISSUE THREAD

**Issue by luzpaz** [SuzanneSoy/SearchBar#29]
![image](https://user-images.githubusercontent.com/4140247/231786203-b33b0c8b-6023-4696-8a3a-98af5feb08a5.png)

source: https://twitter.com/lgworld/status/1646397984538607616?cxt=HHwWgIC-tc6Pl9ktAAAA

This feature would help educate users to find where commands are in the UI

**Comment by SuzanneSoy** [comment]
Good idea. The search results are already grouped in a tree based on toolbars, and the info on toolbars say which workbenches have that toolbar enabled, but a menu bar path would be helpful too!
